server/events/project_command_builder_test.go

```diff
@@ -350,6 +350,58 @@ projects:
 			ExpWorkspace: "myworkspace",
 			ExpDir:       ".",
 		},
+		{
+			Description: "atlantis.yaml with projectname",
```

Comment:
The `Description` should be unique between test cases. It gets used in `t.Run` to identify the name of the test. If a test fails, you look back up to find a test that matches that description. This is a duplicate.

You could say something like `atlantis.yaml with mergeable apply requirement`